src/Core/System/SalesChannel/SalesChannelException.php

```diff
@@ -234,4 +235,32 @@ public static function shippingMethodNotFound(string $shippingMethodId): self
             ['entity' => 'shipping method', 'field' => 'id', 'value' => $shippingMethodId]
         );
     }
+
+    public static function encodingInvalidStructException(string $context): self|\RuntimeException
+    {
+        if (!Feature::isActive('v6.7.0.0')) {
+            return new \RuntimeException($context);
```

Comment:
I am not sure, if we really need that. as the domain exception is also a `\RuntimeException` deep down. So catching this will still work